lib/matplotlib/collections.py

```diff
@@ -545,9 +545,11 @@ def set_offsets(self, offsets):
         offsets = np.asanyarray(offsets)
         if offsets.shape == (2,):  # Broadcast (2,) -> (1, 2) but nothing else.
             offsets = offsets[None, :]
-        self._offsets = np.column_stack(
-            (np.asarray(self.convert_xunits(offsets[:, 0]), float),
-             np.asarray(self.convert_yunits(offsets[:, 1]), float)))
+        cstack = (np.ma.column_stack if isinstance(offsets, np.ma.MaskedArray)
+                  else np.column_stack)
+        self._offsets = cstack(
+            (np.asanyarray(self.convert_xunits(offsets[:, 0]), float),
+                np.asanyarray(self.convert_yunits(offsets[:, 1]), float)))
```

Comment:
```suggestion
             np.asanyarray(self.convert_yunits(offsets[:, 1]), float)))
```
This looks like it is indented too much, I guess the linters don't care either way though because it was previously indented with the parentheses and now it is beyond and both passed.